app/services/config_downloader.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# Service for dowloading SSG configuration files
+class ConfigDownloader
```

Comment:
Observation: This is SSG specific: maybe `SSGConfigDownloader`?

Reply:
Yeah, I guess so :laughing: I initially thought it'd be useful for other configs, but I can't imagine what others we'd need other than SSG information.

Reply:
Updated to `SsgConfigDownloader`.